qldbdriver/qldbdriver.go

```diff
@@ -184,33 +183,33 @@ func (driver *QLDBDriver) GetTableNames(ctx context.Context) ([]string, error) {
 
 // Close the driver, cleaning up allocated resources.
 func (driver *QLDBDriver) Close(ctx context.Context) {
-	if driver.isClosed == false {
+	if !driver.isClosed {
 		driver.isClosed = true
 		for len(driver.sessionPool) > 0 {
 			session := <-driver.sessionPool
 			err := session.endSession(ctx)
 			if err != nil {
-				driver.logger.log(fmt.Sprint("Encountered error trying to end session ", err), LogDebug)
+				driver.logger.logf(LogDebug, "Encountered error trying to end session: '%v'", err.Error())
 			}
 		}
 		close(driver.sessionPool)
 	}
 }
 
 func (driver *QLDBDriver) getSession(ctx context.Context) (*session, error) {
-	driver.logger.log(fmt.Sprint("Getting session. Existing sessions available:", len(driver.sessionPool)), LogDebug)
+	driver.logger.logf(LogDebug, "Getting session. Existing sessions available: %v", len(driver.sessionPool))
```

Comment:
This seems inconsistent again. Can you double check the single quoting of logging in this pr

Reply:
No single quotes here because it's just a number.